### PR TITLE
Toaster revamp

### PR DESCRIPTION
**Pull request by ramshorst**

### CODE REVIEW

**Comment by ramshorst** [comment]
screen shot 2018-06-14 at 16 44 51

File: less/base/_toastr.less [hunk 1]
```diff
@@ -1,48 +1,116 @@
...
-    box-shadow: none;
-    border: 1px solid @upf-primary-white;
-    border-radius: @default-radius;
-    padding: 15px 15px 15px 15px;
-    background-image: none !important;
-    opacity: 0.95;
-
-    &:hover {
+#toast-container {
+  margin-bottom: 16px;
```
Comment (phndiaye) [inline less/base/_toastr.less]: 16px => `spacing-xxx-sm` ?

File: less/base/_toastr.less [hunk 2]
```diff
@@ -1,48 +1,116 @@
...
+        font-weight: normal;
+        font-size: 18/@rem;
+        padding: @spacing-xxx-sm @spacing-xx-sm @spacing-xxx-sm 0;
+        display: flex;
+        align-items: center;
+
+        .toast-title__avatar {
+          width: @avatar-thumbnail;
+          height: @avatar-thumbnail;
+          border-radius: @avatar-thumbnail / 2;
```
Comment (phndiaye) [inline less/base/_toastr.less]: You can replace the 3 precedent lines with `.upf-image--round-36`

File: less/base/_toastr.less [hunk 3]
```diff
@@ -1,48 +1,116 @@
...
+        font-weight: normal;
+        color: inherit;
+        position: initial;
+      }
     }
 
-    .toast-message {
-    	opacity: 0.75;
+    .toast-error {
+      background-color: #f8d7da;
```
Comment (phndiaye) [inline less/base/_toastr.less]: Why not use the `upf-color-feedback-error`, and maybe change its color if it's not suitable anymore ?

File: less/base/_toastr.less [hunk 4]
```diff
@@ -1,48 +1,116 @@
...
       }
     }
-  }
 
-  .toast-error {
-    background-color: @upf-feedback-error;
-    background-image: none;
-  }
+    .toast-warning {
+      background-color: #fff3cd;
```
Comment (phndiaye) [inline less/base/_toastr.less]: same here, we have feedback colors variables :/

File: less/base/_toastr.less [hunk 5]
```diff
@@ -1,48 +1,116 @@
...
+
+      .toast-message a {
+        color: #721c24;
+      }
     }
 
-    a {
-      text-decoration: underline;
+    .toast-success {
+      background-color: #d4edda;
```
Comment (phndiaye) [inline less/base/_toastr.less]: same here, we have feedback colors variables :/

**Comment by phndiaye** [comment]
LGTM 👍

**Pull request merged**